### ISSUE TITLE
QID 376160 not logged in Qualys console

### ISSUE THREAD

**Issue by ufxl070** [Qualys/log4jscanwin#3]
Hello Qualys team,

We performed some   **Log4jScanner.exe /scan /report_sig**  with LOG4J vulnerable detections but after running Qualys agent scans,  in the console, i do not have QID376160 listed despite detections from the utility.

Any recommandations?

Thank you

**Comment by romw** [comment]
Which POD is your agents connected to?  I need to double check on the status of the VM manifest deployment for that POD.

**Comment by MilesinCSAA** [comment]
We are also seeing this issue. any kind of fix? I think our POD is us3.

**Comment by ufxl070** [comment]
Don't know for the POD used...

**Comment by borom** [comment]
We are seeing the same issue. We are on POD 2.

**Comment by MilesinCSAA** [comment]
@romw see you added a comment and code change that might address this issue. should we re-run the script and scan for the QID?

**Comment by romw** [comment]
@MilesinCSAA I've built a new version of the tool, I'm waiting on one of our internal teams to code-sign the tool so Windows Defender won't go crazy when people try to use the utility.  Hopefully I'll have an update posted in a couple of hours.

**Comment by MilesinCSAA** [comment]
Thank you so much for your support Rom.

From: Rom Walton ***@***.***>
Sent: Friday, December 17, 2021 4:56 PM
To: Qualys/log4jscanwin ***@***.***>
Cc: Rogers Jr, Miles ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [Qualys/log4jscanwin] QID 376160 not logged in Qualys console (Issue #3)

Caution: This email originated from outside the company. Do not click links, open attachments, or forward unless you recognize the sender and verify the content is safe. Please report anything suspicious using the Report Phishing button or call the IT Service Desk.


@MilesinCSAA<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMilesinCSAA&data=04%7C01%7Cmiles.rogers%40csaa.com%7Cbc53d75993b5468a3a5008d9c1b8d55b%7C1345b410969446f39c42132a582646b5%7C0%7C0%7C637753821879965248%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=jKrQ6f03%2FXWxeOs1AiTiRuIN7loC7qADgywvM8VTHz8%3D&reserved=0> I've built a new version of the tool, I'm waiting on one of our internal teams to code-sign the tool so Windows Defender won't go crazy when people try to use the utility. Hopefully I'll have an update posted in a couple of hours.

-
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FQualys%2Flog4jscanwin%2Fissues%2F3%23issuecomment-997097232&data=04%7C01%7Cmiles.rogers%40csaa.com%7Cbc53d75993b5468a3a5008d9c1b8d55b%7C1345b410969446f39c42132a582646b5%7C0%7C0%7C637753821880121465%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=%2FWXfwy382OLHFTw7TgRFn5wF2FTacsAoebnuyA4HPeY%3D&reserved=0>, or unsubscribe<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAW62I4FHEAGACA5JRFLX253URPE2RANCNFSM5KJB4KBQ&data=04%7C01%7Cmiles.rogers%40csaa.com%7Cbc53d75993b5468a3a5008d9c1b8d55b%7C1345b410969446f39c42132a582646b5%7C0%7C0%7C637753821880121465%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=%2FtB2vhyL9rSOC7Rvy1R2kDV0zGFTQsdLiM93gDl1Avc%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cmiles.rogers%40csaa.com%7Cbc53d75993b5468a3a5008d9c1b8d55b%7C1345b410969446f39c42132a582646b5%7C0%7C0%7C637753821880121465%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=m6fRNiGwzGaaeZ98PY6Nn4kRxvbU9XWgP%2F78Puyn2YM%3D&reserved=0> or Android<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cmiles.rogers%40csaa.com%7Cbc53d75993b5468a3a5008d9c1b8d55b%7C1345b410969446f39c42132a582646b5%7C0%7C0%7C637753821880121465%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=vvBxwQJ3lr0CL8gtHbY1N8CAK7LMtaNHzP2LXEA5FRc%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

This message may contain information, including personally identifiable information that is confidential, privileged, or otherwise legally protected. If you are not the intended recipient, please immediately notify the sender and delete this message without copying, disclosing, or distributing it.

**Comment by surbo** [comment]
> @romw I've built a new version of the tool, I'm waiting on one of our internal teams to code-sign the tool so Windows Defender won't go crazy when people try to use the utility. Hopefully I'll have an update posted in a couple of hours.

Romw-
Any update on getting the package pushed out?

**Comment by johnmccash** [comment]
Is that why the 1.2.17 prerelease package doesn't have a compiled exe in it?

**Comment by johnmccash** [comment]
Any possibility of getting this soon, or will we all have to wait for Monday? Is the team that handles code-signing available over the weekend?

**Comment by romw** [comment]
A new build is up with a possible fix for this issue when ML is in use.

See:
https://github.com/Qualys/log4jscanwin/releases/download/1.2.17/Log4jScanner-1.2.17.zip

Sorry about the wait, our code-signing cert vendor is having some hiccups and we were having to jump through a few hoops.  I suspect it has been a stressful week across the industry.

Please create a new ticket if this doesn't resolve the issue for any one of you.

Thanks in advance.
